src/Sarif.UnitTests/Transformers/SarifVersionOneToCurrentTests.cs

```diff
@@ -1,16 +1,41 @@
 ﻿// Copyright (c) Microsoft. All rights reserved. Licensed under the MIT        
 // license. See LICENSE file in the project root for full license information.
 
+using System.IO;
+using System.Reflection;
 using FluentAssertions;
 using Microsoft.CodeAnalysis.Sarif.VersionOne;
 using Microsoft.CodeAnalysis.Sarif.Visitors;
 using Newtonsoft.Json;
 using Xunit;
 
-namespace Microsoft.CodeAnalysis.Sarif
+namespace Microsoft.CodeAnalysis.Sarif.UnitTests.Transformers
 {
     public class SarifVersionOneToCurrentTests
     {
+        private static readonly Assembly ThisAssembly;
+        private static readonly string OutputFolderPath;
+        private const string TestLogResourceNameRoot = "Microsoft.CodeAnalysis.Sarif.UnitTests.Transformers.TestLogs";
+
+        static SarifVersionOneToCurrentTests()
+        {
+            ThisAssembly = Assembly.GetExecutingAssembly();
+            OutputFolderPath = Path.Combine(Directory.GetParent(Path.GetDirectoryName(ThisAssembly.Location)).FullName, "UnitTestOutput");
```

Comment:
>GetParent [](start = 54, length = 9)

Is that what you want? The assembly lives at (for example) `C:\Code\sarif\sdk\bld\bin\Sarif.UnitTests\AnyCPU_Release\net461`. So I'd expect you to put the output under `C:\Code\sarif\sdk\bld\bin\Sarif.UnitTests\AnyCPU_Release\net461\UnitTestOutput`. But you put it under `C:\Code\sarif\sdk\bld\bin\Sarif.UnitTests\AnyCPU_Release\UnitTestOutput`. If in future the `net461`-based tests and the `netstandard2.0`-based tests ran simultaneously, they'd interfere with each other.